src/vm/nativeformatreader.h

```diff
@@ -10,6 +10,24 @@
 
 #pragma once
 
+#ifndef DACCESS_COMPILE
+
+#if defined(_M_X64) || defined(__x86_64__) || defined(__i386__) || defined(_M_IX86)
```

Comment:
No, because these aren't about the target architecture. They are about the compilation architecture. I could not find a set of compilation architecture #defines to use.

Reply:
Ah, right. Then you can use `_ARM_`, `_ARM64_`, `_X86_` and `_AMD64_`